aries-site/src/components/content/BulletedList.js

```diff
@@ -6,7 +6,9 @@ export const BulletedList = ({ items }) => {
   return (
     <>
       {items.map(item => (
-        <Text margin={{ bottom: 'xsmall' }}>— {item}</Text>
+        <Text margin={{ bottom: 'xsmall' }} key={item}>
+          — {item}
```

Comment:
I think we need an extra space between - and {.  I believe the design shows the spacing to be equivalent to the length of the dash.

Reply:
FYI, in code editor the dash and the space are equivalent, but when rendered in MetricHPE the spacing is off.
![Screen Shot 2019-11-22 at 2 33 40 PM](https://user-images.githubusercontent.com/1756948/69462029-2eee0500-0d35-11ea-8e77-d084770d96fd.png)

Reply:
Actually, disregard this for now. Sketch files are inconsistent in this treatment. The Invision execution has the single space, but some of the Sketch files show the double space...